doc/source/installation/apt_repo.rst

```diff
@@ -53,171 +53,200 @@ Installing |Percona Server| from Percona ``apt`` repository
+   packages will need to be installed before you can manually install Percona
+   Server: ``mysql-common``, ``libjemalloc1``, ``libaio1`` and ``libmecab2``
 
 
 Running |Percona Server|
 ========================
 
-|Percona Server| stores the data files in :file:`/var/lib/mysql/` by default. You can find the configuration file that is used to manage |Percona Server| in :file:`/etc/mysql/my.cnf`. 
+|Percona Server| stores the data files in :file:`/var/lib/mysql/` by
+default. You can find the configuration file that is used to manage |Percona
+Server| in :file:`/etc/mysql/my.cnf`.
 
 .. note:: 
 
-  *Debian* and *Ubuntu* installation doesn't automatically create a special ``debian-sys-maint`` user which can be used by the control scripts to control the |Percona Server| ``mysqld`` and ``mysqld_safe`` services like it was the case with previous |Percona Server| versions. If you still require this user you'll need to create it manually.
-
-1. Starting the service
+   *Debian* and *Ubuntu* installation doesn't automatically create a special
+    ``debian-sys-maint`` user which can be used by the control scripts to
+    control the |Percona Server| ``mysqld`` and ``mysqld_safe`` services like it
+    was the case with previous |Percona Server| versions. If you still require
+    this user you'll need to create it manually.
 
-   |Percona Server| is started automatically after it gets installed unless it encounters errors during the installation process. You can also manually start it by running: 
-
-   .. code-block:: bash
+|tip.run-all.root|
 
-     $ sudo service mysql start
-
-2. Confirming that service is running 
-
-   You can check the service status by running:  
-
-   .. code-block:: bash
-
-     $ service mysql status
-
-3. Stopping the service
-
-   You can stop the service by running:
-
-   .. code-block:: bash
+1. Starting the service
 
-     $ sudo service mysql stop
+   |Percona Server| is started automatically after it gets installed unless it
+   encounters errors during the installation process. You can also manually
+   start it by running: :bash:`service mysql start`
 
-4. Restarting the service 
+#. Confirming that service is running. You can check the service status by
+   running: :bash:`service mysql status`
 
-   You can restart the service by running: 
+#. Stopping the service
 
-   .. code-block:: bash
+   You can stop the service by running: :bash:`service mysql stop`
 
-     $ sudo service mysql restart
+#. Restarting the service. :bash:`service mysql restart`
 
 .. note:: 
 
-  *Debian* 8.0 (jessie) and *Ubuntu* 15.04 (vivid) come with `systemd <http://freedesktop.org/wiki/Software/systemd/>`_ as the default system and service manager so you can invoke all the above commands with ``sytemctl`` instead of ``service``. Currently both are supported.
+   Debian 8.0 (jessie) and Ubuntu 15.04 (vivid) come with `systemd
+   <http://freedesktop.org/wiki/Software/systemd/>`_ as the default system and
```

Comment:
Both of these aren't supported. Maybe use stretch/bionic instead or just drop the distro names.